CONTRIBUTING.md

```diff
@@ -151,6 +151,36 @@ Running the script for a project in `sdk\tables` would look like this:
 eng\scripts\Export-API.ps1 tables
 ```
 
+## Updating Sample Snippets
+If the specific client library has sample snippets in markdown format, they were most likely created with help of the `eng\scripts\Update-Snippets.ps1` script.
+Any changes made to the snippet markdown should be done via updating the corresponding C# snippet code and subsequently running the script.
+
+Running the script for a project, for example in `sdk\keyvault`, would look like this: 
+```
+eng\scripts\Update-Snippets.ps1 keyvault
```

Comment:
Does this imply that snippet names need to be globally unique under a given service directory?

Reply:
I assume so, but @pakrym could confirm.

Reply:
Yes - https://github.com/Azure/azure-sdk-for-net/blob/master/eng/SnippetGenerator/Program.cs#L21

Reply:
That would probably be a helpful thing to explicitly call out.